PYTHON/DEMOS/SISO_RX.py

```diff
@@ -322,6 +322,9 @@ def animate(i, num_samps, recorder, agc_en, wait_trigger, info):
 
     # Store received samples in binary file (second method of storage)
     write_to_file('./data_out/rxsamps', sampsRx)
+    #mdic = {"rxdata": sampsRx[0]}
+    #savemat("data_rmode2.mat", mdic)
```

Comment:
Remove commented code

Reply:
useful for testing things, I'll try to make it prettier on the next commit